arango/resolver.py

```diff
@@ -7,20 +7,24 @@
 
 import random
 from abc import ABC, abstractmethod
+from typing import Optional, Set
 
 
 class HostResolver(ABC):  # pragma: no cover
     """Abstract base class for host resolvers."""
 
+    def __init__(self) -> None:
+        self._max_tries: int = 3
```

Comment:
Should this value be user-configurable?

Reply:
Good call, [it is now](https://github.com/ArangoDB-Community/python-arango/pull/184/files#diff-d72fa74f337fd1eb022878d3e3f0f8889bfa8872a933a67e851f37c9160d535dR34-R37)